impl/src/main/java/jakarta/faces/convert/EnumConverter.java

```diff
@@ -66,27 +69,6 @@ public class EnumConverter implements Converter, PartialStateHolder {
      */
     public static final String ENUM_NO_CLASS_ID = "jakarta.faces.converter.EnumConverter.ENUM_NO_CLASS";
 
-    private Class<? extends Enum> targetClass;
-    private boolean isTransient;
-    private boolean initialState;
-
-   /**
-    *  For StateHolder
-    */
-    public EnumConverter() {
-
-    }
-
-    /**
-     * Instantiates an enum converter with a class where enum constants are taken from.
-     *
-     * @param targetClass Class where the enum constants are taken from by the converter methods.
-     */
-    public EnumConverter(Class targetClass) {
-        this.targetClass = targetClass;
-    }
```

Comment:
This is a direct API change. This is not allowed. You should create a ticket in faces project and discuss why this should be removed and then create PR to mark it deprecated and then only remove it in a next major version.

Please revert.

Reply:
ouch ... ok!
... I though it was an implementation choice due to old EL/Java version ...